### ISSUE TITLE
did mpv get broken in last upgrade?

### ISSUE THREAD

**Issue by notevenodd** [termux/termux-packages#2750]
when launching mpv, I'm getting this:

```
WARNING: linker: Unsupported flags DT_FLAGS_1=0x8
WARNING: linker: Unsupported flags DT_FLAGS_1=0x8
CANNOT LINK EXECUTABLE DEPENDENCIES: cannot locate symbol "__register_atfork" referenced by "libpulsecommon-12.2.so"...
```

**Comment by its-pointless** [comment]
This is possy due to pulseaudio using ad different version of platform libs.

**Comment by tomty89** [comment]
Yeah looks like header guard is not enough. Probably should have copied libaaudio alone as you suggested. @its-pointless

**Comment by fornwall** [comment]
Thanks for reporting!

This should be fixed in the latest 12.2-6 version of the libpulseaudio package - run `pkg up` to update.
